src/Expensify.js

```diff
@@ -79,6 +83,7 @@ const defaultProps = {
     },
     updateAvailable: false,
     initialReportDataLoaded: false,
+    betas: null,
```

Comment:
Should this be `[]` since we're telling PropTypes it should be an array?

Also it looks like we default betas to `[]` in a few places already like `AuthScreens.js` and `NewWorkspacePage.js`

Reply:
Well, we're telling proptypes that the component receives an array and nothing else for betas. But it's not a required param so within the component if we've not received anything we're defaulting to null and that's fine. I think having this as an empty array or null are both fine. Having it null is just simpler on the conditional checks so I went ahead with that. Plus I think null is better for my scenario because what if the betas are loaded and they are empty. At this point my code will go ahead and call Navigation.navigate which is what I want.

Reply:
> Plus I think null is better for my scenario because what if the betas are loaded and they are empty

This is a good point, it's definitely possible for someone to get betas loaded and they don't have access to any betas, which I assume would lead to `betas` being `[]`. If the only use of this data is to check if betas have loaded or not, I think it makes to not default to `[]` 👍

Reply:
Still NAB: But I think this should be `[]`. I think that more closely fits the proptypes which is `PropTypes.arrayOf(PropTypes.string)`.

Reply:
Discussing this on slack. I know generally for react if the proptype is not required then its totally okay to use undefined or null in the defaultProp ([example](https://stackoverflow.com/questions/37842868/how-can-i-declare-a-proptype-corresponding-to-a-nullable-number)). Was wondering if Expensify has a different style choice so having this discussion on [slack](https://expensify.slack.com/archives/C03TQ48KC/p1627325849151800).

Reply:
Updated